### PR TITLE
Update dependencies

### PR DESCRIPTION
**Pull request by minwoox**
- Armeria 0.99.9 -> 1.0.0
- Micrometer 1.5.3 -> 1.5.4
- Spring Boot 2.3.2.RELEASE -> 2.3.3.RELEASE
- Build
  - AssertJ 3.16.1 -> 3.17.0
  - Gradle 6.5.1 -> 6.6.1
  - JMH 1.23 -> 1.25
  - Mockito 3.4.6 -> 3.5.7

### CODE REVIEW

File: dependencies.yml [hunk 1]
```diff
@@ -4,8 +4,8 @@
 #
 boms:
 - com.fasterxml.jackson:jackson-bom:2.11.2
-- com.linecorp.armeria:armeria-bom:0.99.9
-- io.micrometer:micrometer-bom:1.5.3
+- com.linecorp.armeria:armeria-bom:1.0.0
```
Comment (ikhoon) [inline dependencies.yml:7]: Update Javadoc link to `https://www.javadoc.io/doc/com.linecorp.armeria/armeria-javadoc/1.0.0/`?

**Review by ikhoon — APPROVED** [review]
Thanks! @minwoox

File: dependencies.yml (hunk 1 above)
Comment (minwoox) [reply dependencies.yml:7]: Oops Thanks!

**Comment by minwoox** [comment]
Thanks for reviewing!

**Pull request merged**